changelog/1.12/DCOS-21451.bugfix

```diff
@@ -0,0 +1,2 @@
+Prevent reusing tcp sockets by AR's cache code. Fixes the bug when Marathon UI
```

Comment:
These are example changelog news-fragment for the changes that went into this train - https://github.com/dcos/dcos/pull/2558. 
Each news-fragment should have been included in the constituent PR.